root/entity/Subscribers.js

```diff
@@ -76,32 +116,68 @@ const Subscribers = ({
           </ul>
         </>
       ) : (
-        <p>
-          {texp.l('There are currently no users subscribed to {entity}.',
-            {entity: entity.name})}
-        </p>
+        entityType === 'editor' ? (
+          viewingOwnProfile ? (
+            <p>
+              {l(`There are currently no users subscribed to edits
+                  that you make.`)}
+            </p>
+          ) : (
+            <p>
+              {texp.l(`There are currently no users subscribed to edits
+                       that {user} makes.`,
+                      {user: entity.name})}
+            </p>
+          )
+        ) : (
+          <p>
+            {texp.l('There are currently no users subscribed to {entity}.',
+                    {entity: entity.name})}
+          </p>
+        )
       )}
 
-      {subscribed ? (
-        <p>
-          {exp.l('You are currently subscribed. {unsub|Unsubscribe}?',
-            {unsub: '/account/subscriptions/' + entityType + '/remove?id=' + entity.id})}
-        </p>
-      ) : (
-        (publicEditors.length + privateEditors === 0) ? (
+      {viewingOwnProfile ? null : (
+        subscribed ? (
           <p>
-            {exp.l('Be the first! {sub|Subscribe}?',
-              {sub: '/account/subscriptions/' + entityType + '/add?id=' + entity.id})}
+            {exp.l('You are currently subscribed. {unsub|Unsubscribe}?',
+                   {
+                     unsub: '/account/subscriptions/' +
+                            entityType +
+                            '/remove?id=' +
+                            entity.id,
```

Comment:
Might it make sense to build these links above instead? e.g.

```JavaScript
const unsubLink = '/account/subscriptions/' + entityType +
  '/remove?id=' + entity.id;

// ...

{exp.l('You are currently subscribed. {unsub|Unsubscribe}?',
       {unsub: unsubLink})}
```

('Cause while the old version slightly exceeded 78 chars, I think blowing it up like this inline made it look a bit busier than it needs to.)